src/offline/cable_output.F90

```diff
@@ -40,15 +40,15 @@ MODULE cable_output_module
   USE cable_def_types_mod
   USE casavariable, ONLY: casa_pool, casa_flux, casa_met
   USE cable_IO_vars_module
-  USE cable_checks_module, ONLY: mass_balance, energy_balance, ranges
+  USE cable_checks_module, ONLY: mass_balance, energy_balance, ranges, check_range
   USE cable_write_module
   USE netcdf
   USE cable_common_module, ONLY: filename, calcsoilalbedo, CurYear,IS_LEAPYEAR, cable_user,&
        gw_params
   IMPLICIT NONE
   PRIVATE
-  PUBLIC open_output_file, write_output, close_output_file, create_restart
-  INTEGER :: ncid_out ! output data netcdf file ID
+  PUBLIC open_output_file, write_output, close_output_file, create_restart, check_and_write
+  INTEGER, SAVE :: ncid_out, ncid_restart ! output/restart data netcdf file ID
```

Comment:
Why add the SAVE keyword? All the entries in the MODULE specification will be saved by default.

Reply:
I also got confused by `SAVE` in the `out` variable, should I remove the attribute there as well?

Reply:
For a link about the SAVE statement: https://community.intel.com/t5/Intel-Fortran-Compiler/Placement-of-SAVE-statement-within-a-module/td-p/1154139

I'm not sure what SAVE statement you are talking about but I wouldn't use SAVE in any module specification area since the standards now say it isn't required.